tools/scripts/compile.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+compile()
+{
+    local version="${1:-$(date -u)}"
```

Comment:
not sure about the default version number being the date, maybe some kind of placeholder to indicate that it's just a nightly-style build